include/OpenColorIO/OpenColorTypes.h

```diff
@@ -776,10 +775,7 @@ extern OCIOEXPORT const char * OCIO_OPTIMIZATION_FLAGS_ENVVAR;
  */
 extern OCIOEXPORT const char * OCIO_USER_CATEGORIES_ENVVAR;
 
-/** @}*/
-
-/** \defgroup VarsRoles
- *  @{
+/** VarsRoles
```

Comment:
Minor:  I suspect "VarsRoles" was a name for the group, but it doesn't make sense as a description of the code.  Same for the next few groups.  These sections already have an rst header, so it might be better to just remove all three doxygen group lines.